### PR TITLE
Fix/scenario1 5 new test

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Moved test script to bastion machine

### CODE REVIEW

File: playbooks/scenario1_5_setup.yml [hunk 1]
```diff
@@ -51,3 +58,85 @@
...
+
+- name: Copy and configure test script files to Bastion
+  hosts: bastion
+  vars:
+    test_path: /home/linux/test
+    project_path: /home/ansideln/work/customer-service-monitoring
+    keys_path: "{{ project_path }}/scenarios/scenario1_5"
+    inventory_path: "{{ test_path }}/inventory/prod"
+    loadbalancer_public_ip: "{{ lookup('env', 'LOADBALANCER_PUBLIC_IP') }}"
+    verion: "HEAD"
```
Comment (lego963) [inline playbooks/scenario1_5_setup.yml]: is it ok `verion` ?

**Comment by lego963** [comment]
endlines
scenarios/scenario1_5/test/main.sh → playbooks/files/lb_down_test.sh
playbooks/scenario1_5_start_server.yml

File: playbooks/scenario1_5_setup.yml (hunk 1 above)
Comment (anton-sidelnikov) [reply playbooks/scenario1_5_setup.yml]: yep, pointer to the last commit in branch

Comment (anton-sidelnikov) [reply playbooks/scenario1_5_setup.yml]: fixed

**Pull request merged**